src/app/login/page.tsx

```
@@ -82,38 +130,59 @@ export default function Login() {
+              setLoading(true);
+              setError(null);
+              setSuccessMessage(null);
+
+              // Usar el contexto de autenticación para login
+              try {
+                await authLogin(email, password);
+              } catch (authError) {
+                // Intentar con el AuthService directo como fallback
+                const response = await AuthService.login({ email, password });
+                if (response && response.user && response.user.id) {
+                  // Asegurarse de que el ID del usuario se guarde correctamente
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)